ext/opentelemetry-ext-docker-tests/tests/pymysql/test_pymysql_functional.py

```diff
@@ -81,7 +81,7 @@ def test_executemany(self):
         """Should create a child span for executemany
         """
         with self._tracer.start_as_current_span("rootSpan"):
-            data = ["1", "2", "3"]
+            data = (("1",), ("2",), ("3",))
```

Comment:
I think these changes can be removed from this PR

Reply:
I'll merge master and these changes should be done.